Formula/liquigraph.rb

```diff
@@ -0,0 +1,38 @@
+class Liquigraph < Formula
+  desc "Migration runner for Neo4j"
+  homepage "http://www.liquigraph.org"
+  url "https://github.com/fbiville/liquigraph/archive/liquigraph-3.0.0.tar.gz"
+  sha256 "4864c323a626c15df9fed49d05ede3e947e0b78fe864ff81dcc8703b875662b1"
+  head "https://github.com/fbiville/liquigraph.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.8+"
+
+  def install
+    ENV.java_cache
+    system "mvn", "-q", "clean", "package", "-DskipTests"
+    mkdir "binaries"
```

Comment:
Are there other issues I need to deal with? Or is this mergeable? :-)
https://github.com/Homebrew/homebrew-core/pull/509#issuecomment-213300733 basically said we could ignore the notability audit.